routing/missioncontrol.go

```diff
@@ -125,9 +125,9 @@ type TimedPairResult struct {
 	// timestamp is the time when this result was obtained.
 	Timestamp time.Time
 
-	// minPenalizeAmt is the minimum amount for which a penalty should be
-	// applied based on this result. Only applies to fail results.
-	MinPenalizeAmt lnwire.MilliSatoshi
+	// Amt is the amount that was (attempted to be) forwarded in the last
+	// payment attempt.
```

Comment:
Not sure what you mean with 'attempted to be forwarded= failed'. But this is (usually) the amount of the last failure.

Comments fixed to explain this better.

Reply:
Ah, now seeing that this refers to the first commit. Removed the "(attempted to be)" part. It may not be that helpful, because there is also a success boolean in this struct.